drivers/hmis_data_quality_tool/app/models/hmis_data_quality_tool/enrollment.rb

```diff
@@ -620,6 +693,297 @@ def self.sections(report) # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticC
             ! item.enrollment_coc.in?(item.project_coc_codes)
           },
         },
+        hoh_income_percent_ami: {
+          title: 'Head of Household is Missing Income as a Percent of AMI',
+          description: 'Income as a Percent of AMI is missing or not collected',
+          required_for: 'HoH in HP or RRH',
+          detail_columns: default_detail_columns + [
+            :percent_ami,
+          ],
+          denominator: ->(item) {
+            funding_sources = [
+              HudUtility2024.funder_components['VA: SSVF'],
+            ].flatten
+            project_types = [
+              HudUtility2024.performance_reporting[:rrh],
+              HudUtility2024.performance_reporting[:prevention],
+            ].flatten
+
+            # Only hohs that include the above funder(s) and project type(s)
+            hoh?(item) && ! (funding_sources & item.funders).empty? && project_types.include?(item.project_type)
```

Comment:
Is this the same as:
```suggestion
            hoh?(item) && project_types.include?(item.project_type) && (funding_sources & item.funders).any? 
```
Two proposed changes there, one to push the project type check up since I _think_ that's less work for ruby, and the second to use `any?` instead of `! (..).empty?)` just to try and make it a bit more readable.